client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -79,12 +84,24 @@ export class CheckoutThankYouHeader extends PureComponent {
 			hasFailedPurchases,
 			primaryPurchase,
 			displayMode,
+			purchases,
 		} = this.props;
 
 		if ( hasFailedPurchases ) {
 			return translate( 'Some of the items in your cart could not be added.' );
 		}
 
+		if ( purchases && purchases[ 0 ].productType === 'search' ) {
+			return (
+				translate( 'We are currently indexing your site.' ) +
+				'\n' +
+				translate(
+					'In the meantime we have configured Jetpack Search on your site.' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'In the meantime, we have configured Jetpack Search on your site — you should try customizing it in your traditional WordPress dashboard.' )` *ES Score: 7*